etna/feedback/templates/feedback/submit.html

```diff
@@ -0,0 +1,30 @@
+{% extends "base.html" %}
+
+{% load i18n datalayer_tags wagtailcore_tags feedback_tags %}
+
+{% block title %}{% trans "Share your feedback" %}{% endblock %}
+
+{% block extra_gtm_js %}
+    {% render_gtm_datalayer view %}
+{% endblock extra_gtm_js %}
```

Comment:
What is the URL to check the dataLayer for this template?

Reply:
Look for the `action=""` attribute on the response `<form>`... you can visit that URL via a GET request.

NOTE: The conversation about tracking for this process is still 'in progress', so this will be refined at a later time.

Reply:
We can just remove it all together, I think.

Reply:
This is now removed